src/components/Plants/PlantingSiteDetails.tsx

```diff
@@ -67,9 +67,19 @@ export default function PlantingSiteDetails(props: PlantingSiteDetailsProps): JS
             'populations.totalPlants',
           ],
           search: {
-            operation: 'field',
-            field: 'plantingSite_id',
-            values: [plantingSite.id],
+            operation: 'and',
+            children: [
+              {
+                operation: 'field',
+                field: 'plantingSite_id',
+                values: [plantingSite.id],
+              },
+              {
+                operation: 'field',
```

Comment:
Were we showing results from other orgs previously?  😮

Reply:
I don’t think so but added if.

Reply:
i will remove this, we can add it later if needed